src/ast.ts

```diff
@@ -1701,6 +1724,17 @@ export abstract class DeclarationStatement extends Statement {
   isAny(flag: CommonFlags): bool { return (this.flags & flag) != 0; }
   /** Sets a specific flag or flags. */
   set(flag: CommonFlags): void { this.flags |= flag; }
+
+  /** Attempts to return the name as an identifier. */
+  get ident(): IdentifierExpression {
+    assert(this._name.kind == NodeKind.IDENTIFIER || this._name.kind == NodeKind.CONSTRUCTOR);
+    return this._name as IdentifierExpression;
```

Comment:
any better name for this function?

Reply:
Perhaps `identifier`, we don't abbreviate so much elsewhere if I'm not mistaken.